python/nav/web/seeddb/page/netbox/edit.py

```diff
@@ -198,11 +202,7 @@ def snmp_write_test(ip, profile):
     syslocation = '1.3.6.1.2.1.1.6.0'
     value = ''
     try:
-        snmp = Snmp(
-            ip,
-            profile.configuration.get("community"),
-            profile.configuration.get("version"),
-        )
+        snmp = get_snmp_session_for_profile(profile)(ip)
```

Comment:
Did you remember to import/write `get_snmp_session_for_profile`?

Reply:
Yes and not.  Apparently, I've rebased this on the wrong branch after-the-fact, so this function is not actually available.  I'll have to force-push an update

Reply:
Rebased to the correct branch now (where this function was already imported for use in the read profile tests)